src/scripts/components/main/onboarding-tooltip/onboarding-tooltip.tsx

```diff
@@ -68,12 +70,20 @@ const OnboardingTooltip: FunctionComponent<Props> = ({
     const setPosition = () => {
       setReferencePosition(referenceElement.getBoundingClientRect());
     };
-
     window.addEventListener('resize', setPosition);
-
     setReferencePosition(referenceElement.getBoundingClientRect());
 
-    return () => window.removeEventListener('resize', setPosition);
+    // hightlight current ui reference: move element in front of the overlay
+    referenceElement.style.zIndex = '4';
```

Comment:
I would prefer top have these things attached a class in the CSS rather than inline here.